packages/api/controllers/nomination.js

```diff
@@ -90,7 +93,14 @@ const updateNomination = async (req, res) => {
         console.error('Was not able to change reminderSent bool', error)
       }
 
-      if (nomination.status === 'Decline') {
+      if (nomination.status === 'Declined') {
```

Comment:
Do we know for sure whether we want `'Decline'` or `'Declined'` here? I don't know off hand which is correct, but want to ensure we pick the right one.

Additionally, to be safe, I think it may make sense to move these values out to a variable that we can reference. A pattern I've seen work well is to create a const object that would look something like this:

```javascript
const NOMINATION_STATUSES = {
  declined: 'Declined',
  ready: 'Ready for Board Review',
  // ...etc
};
```

This way any time we need set/compare one of these values, we can just reference `NOMINATION_STATUSES.declined` (just using `'Declined'` as an example here) and be confident we're comparing the correct value and not making any typos.

Reply:
It's definitely Declined, it does not register with Decline. I like that pattern.